test-requirements.txt

```diff
@@ -14,13 +14,13 @@ pluggy==1.0.0
     # via pytest
 py==1.11.0
     # via pytest
-pyparsing==3.0.6
+pyparsing==3.0.7
     # via packaging
-pytest==6.2.5
+pytest==7.0.0
     # via -r test-requirements.in
-toml==0.10.2
```

Comment:
TIL

Reply:
Hmm curious, must have been a change between pytest 6 and 7, dont think this will mean anything for confidence though.

Reply:
No idea what the difference is, but decided I don't really care :)